src/components/VmActions/ConsoleNotificationsDialog.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react'
+
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { withMsg } from '_/intl'
+import ConfirmationModal from './ConfirmationModal'
+
+import * as Actions from '_/actions/console'
+
+import {
+  CONSOLE_IN_USE,
+  CONSOLE_LOGON,
+} from '_/constants'
+
+import {
+  BROWSER_VNC,
+  NATIVE_VNC,
+  SPICE,
+} from '_/constants/console'
+
+const ConsoleNotificationsDialog = ({
+  vmId,
+  vmName,
+  consoleType,
+  status,
+  msg,
+  dismissConsoleError,
+  openConsole,
+}) => {
+  const onClose = () => dismissConsoleError({ consoleType, vmId })
```

Comment:
So this is what removes the error from `consoles.error` to allow multiple errors to be displayed?